src/main/java/org/springframework/data/hazelcast/repository/config/Constants.java

```diff
@@ -16,8 +16,12 @@
 package org.springframework.data.hazelcast.repository.config;
 
 /**
- * Constants used in this project
+ * Constants used in this project.
+ *
+ * @deprecated Don't use these constants, since it's not guaranteed that a HazelcastInstace with the name {@literal
```

Comment:
minor: maybe explicitly mention that we'll remove this in a future release

Reply:
Added, thanks for the comment.